src/wallet/wallet.cpp

```diff
@@ -127,7 +127,10 @@ SaplingPaymentAddress CWallet::GenerateNewSaplingZKey()
     libzcash::SaplingExtendedSpendingKey xsk;
     do
     {
+        // always derive hardened keys
```

Comment:
Remove this comment (wasn't in the commit, and doesn't apply to us in the way it was intended upstream).